e107_plugins/forum/templates/forum_viewtopic_template.php

```diff
@@ -481,6 +481,14 @@
 
 									";
 
+$FORUM_VIEWTOPIC_TEMPLATE['quickreply'] = "
+						<div class='form-group'>
+							<textarea cols='80' placeholder='{LAN=FORUM_2007}' rows='4' id='forum-quickreply-text' class='tbox input-xxlarge form-control' name='post' onselect='storeCaret(this);' onclick='storeCaret(this);' onkeyup='storeCaret(this);'></textarea>
```

Comment:
textarea should also be in a shortcode. The placeholder can be the default LAN, unless the shortcode receives a parm called 'label'

Reply:
Ok